src/happy_vllm/model/model_base.py

```diff
@@ -51,23 +54,24 @@ def __init__(self, **kwargs):
         self._tokenizer = None
         self._model_conf = None
         self._model_explainer = None
+        self.openai_serving_tokenization = None
```

Comment:
mypy error because this attribute isn't initiated in __init__ and mypy selects the first type found in the class.

I will add the initiation of the other attributes to None